tests/ui/macros/missing-writer.rs

```diff
@@ -0,0 +1,11 @@
+//Check error for missing writer in writeln! and write! macro
```

Comment:
Not important, but comments generally have a space before text, so it's easier to read
```suggestion
// Check error for missing writer in writeln! and write! macro
```